src/Storefront/Resources/app/storefront/src/plugin/forms/form-submit-loader.plugin.js

```diff
@@ -60,7 +60,7 @@ export default class FormSubmitLoaderPlugin extends Plugin {
         const id = this._form.id;
         if (!id) return false;
 
-        this._submitButton = DomAccess.querySelector(this._form, `button[type=submit][form=${id}]`);
+        this._submitButton = DomAccess.querySelector(document.body, `button[type=submit][form=${id}]`, false);
```

Comment:
Why did you changed the first parameter?

Reply:
I think this is reasonable, since the button can be outside of the form, when specifying the `form`-attribute:
https://www.w3schools.com/tags/att_button_form.asp

It is quite handy in different places, and we stumbled upon this issue as well.

Reply:
I just noticed that there can in fact be multiple submit buttons, which are also not covered by the current code. But maybe this is for another PR :)

Reply:
As @aragon999 said: you use it to connect a button outside the form to it. So I need to look outside the form. We did. Was nice to easily re-arrange the layout. We were fine with dropping the loading for it in that moment.

Reply:
In general this seems reasonable for layouts where the button is somewhere else.

The only concern I have is, that the selector now grabs all buttons with the matching ID in the body. What would happen if there are for some reason two forms on the same page with the same ID (due to a copy-paste error, custom CMS stuff, template extensions or whatnot). Would it then submit the first found button (because of querySelector behavior) - even if I clicked another submit button?

Normally I would think that this is not an issue because there should be no duplicate IDs in the first place but maybe there already are cases out there which will now suddenly have different behavior.

Maybe it would be viable to add an option for this like `formWrapperSelector` to keep the default but make it possible to set it equal to body if needed.

Reply:
Duplicate IDs will lead to invalid HTML, not sure if that should be handled or not.

Probably the HTML should be fixed instead and not give the possibility to work around this issue.

Maybe one could add something like `:not(form)` in front of the selector to not select buttons which are contained within other forms, although I think this would be still a valid case (although I can never remember that I used such construct for something reasonable).

Reply:
Sure this is invalid HTML, and it should be fixed. But it might already be there in customized environments, and then something would break after the next update, even though I think it is unlikely. The selector option wasn't meant to be a support for duplicate IDs, I just thought of it as a backwards compatibility mechanism to be extra sure. What do you think @JoshuaBehrens ?

I would also be fine with the current implementation I guess, but then I would like to add a hint to the changelog, that you have to verify that no duplicate IDs are present, because the selector is now searching the whole document.

Reply:
I like to be able to change the container. I feel that I will have to PR it later anyway when we talk about it like that. Nevertheless I am with @aragon999 . Maybe we just do both. Add a selector for the nearest parent that defaults to body. AND add a warning when we find two hits with the id.

Reply:
Yes, that sounds as a reasonable implementation, I am fine with that.

I am currently waiting for this PR to be merged to add another PR which makes it also possible to have buttons inside and outside the form simultaneously, which are both set to a loading state.

Reply:
Why do you need to pressure me? :D @aragon999

Reply:
:upside_down_face: sorry, no pressure, it is not that important :)